src/core/stream.h

```diff
@@ -471,6 +489,8 @@ QuicStreamRecvGetState(
 {
     if (Stream->Flags.RemoteNotAllowed) {
         return QUIC_STREAM_RECV_DISABLED;
+    } else if (Stream->Flags.RemoteCloseResetReliable) {
+        return QUIC_STREAM_RECV_RELIABLE_RESET;
     } else if (Stream->Flags.RemoteCloseReset) {
```

Comment:
Does receiving a normal reset override the reliable reset state? If so, then it should be first in the `if`. And we need to make sure our logic elsewhere agrees.

Reply:
I thought initially we couldn't reset, but it looks like reset frames according to the protocol are allowed, I can update.